{{ cookiecutter.repo_name }}/README.md

```diff
@@ -50,30 +61,33 @@ If you not only want to fix the software stack but also the underlying OS, use
 in combination with any of the modes above.
 See the [Snakemake documentation](https://snakemake.readthedocs.io/en/stable/executable.html) for further details.
 
-# Step 4: Investigate results
+### Step 5: Investigate results
 
 After successful execution, you can create a self-contained interactive HTML report with all results via:
 
     snakemake --report report.html
 
 This report can, e.g., be forwarded to your collaborators.
+An example (using some trivial test data) can be seen [here](https://cdn.rawgit.com/snakemake-workflows/rna-seq-kallisto-sleuth/master/.test/report.html).
+
+### Step 6: Commit changes
 
-### Advanced
+Whenever you change something, don't forget to commit the changes back to your github copy of the repository:
 
-The following recipe provides established best practices for running and extending this workflow in a reproducible way.
+    git commit -a
+    git push
 
-1. [Fork](https://help.github.com/en/articles/fork-a-repo) the repo to a personal or lab account.
-2. [Clone](https://help.github.com/en/articles/cloning-a-repository) the fork to the desired working directory for the concrete project/run on your machine.
-3. [Create a new branch](https://git-scm.com/docs/gittutorial#_managing_branches) (the project-branch) within the clone and switch to it. The branch will contain any project-specific modifications (e.g. to configuration, but also to code).
-4. Modify the config, and any necessary sheets (and probably the workflow) as needed.
-5. Commit any changes and push the project-branch to your fork on github.
-6. Run the analysis.
-7. Optional: Merge back any valuable and generalizable changes to the [upstream repo](https://github.com/snakemake-workflows/{{cookiecutter.repo_name}}) via a [**pull request**](https://help.github.com/en/articles/creating-a-pull-request). This would be **greatly appreciated**.
-8. Optional: Push results (plots/tables) to the remote branch on your fork.
-9. Optional: Create a self-contained workflow archive for publication along with the paper (snakemake --archive).
-10. Optional: Delete the local clone/workdir to free space.
+### Step 7: Contribute back
```

Comment:
May I ask you to also include the "update from upstream" step that is described in the dna-seq-varlociraptor workflow?

Reply:
Done, also included an example `.github/workflow/main.yaml` for GitHub Actions.